main.js

```diff
@@ -439,7 +453,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
         });
```

Comment:
Surely all this does is delays adding anything to the queue for 500ms. I think the change of concurrency limit from 5 to 2 will have the effect you're after